addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/discovery/MilightBridgeDiscovery.java

```diff
@@ -52,7 +52,7 @@ private void startDiscoveryService() {
             try {
                 receiveThread = new MilightDiscover(this, 200, 2000 / 200);
             } catch (SocketException e) {
-                logger.error("Opening a socket for the milight discovery service failed. " + e.getLocalizedMessage());
+                logger.error("Opening a socket for the milight discovery service failed. {}", e.getLocalizedMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace